extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -461,6 +462,7 @@ static function packFiles( $fileArray, $subPath = '', $fileExtension = '.js', $p
 
             $content .= "/* start: $file */\r\n";
             $content .= $fileContent;
+            if ($isJS) {$content .= ';';}
```

Comment:
neither, the repo follows the ez coding standard, so it would either be:
```php
if ( $isJS ) $content .= ';';
```

or preferably as it would be consistent with the rest of the code here:
```php
if ( $isJS )
{
    $content .= ';';
}
```